src/py-opentimelineio/opentimelineio/console/otiopluginfo.py

```diff
@@ -185,6 +199,21 @@ def main():
     # load all the otio plugins
     active_plugin_manifest = otio.plugins.ActiveManifest()
 
+    # print version information to the shell
+    if args.version:
+        print("OpenTimelineIO version: {}".format(otio.__version__))
+
+        if pkg_resources:
+            pkg_resource_plugins = list(
+                pkg_resources.iter_entry_points("opentimelineio.plugins")
+            )
+            if pkg_resource_plugins:
+                print("Plugins from pkg_resources:")
+                for plugin in pkg_resource_plugins:
+                    print("   {}".format(plugin.dist))
+            else:
+                print("No pkg_resource plugins installed.")
```

Comment:
Is the intent to not exit after printing version info and continue printing the usual info?

Reply:
Correct.  The thought was that this program is just dumping information about the installation anyway, so `--version` is an additive option and doesn't prevent printing everything else.  Happy to change that if you think its too much of a break from expected behavior though.